fastplotlib/layouts/_figure.py

```diff
@@ -492,6 +530,9 @@ def _render(self, draw=True):
         for subplot in self:
             subplot._render()
 
+        # overlay render pass
+        self.renderer.render(self._overlay_scene, self._overlay_camera, flush=False)
```

Comment:
@almarklein the overlay render pass without the depth buffer doesn't exist yet right? we'll just keep this here for now and update later I guess.